src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start--day--sun calendar--month--length--29">
+      <div class="calendar__day calendar__day--number--1"></div>
+      <div class="calendar__day calendar__day--number--2"></div>
+      <div class="calendar__day calendar__day--number--3"></div>
+      <div class="calendar__day calendar__day--number--4"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```
всі ці додаткові класи не потрібні. В тебе дні мають з'являютися за рахунок циклу